### ISSUE TITLE
NTR: X-ray lateral neck: increased pre-vertebral space (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9019]
New term request X-ray lateral neck: increased pre-vertebral space (nadea3_190206155313):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000290
- label: Prevertebral space enlargement
- parents: [Abnormal pharynx morphology (HP:0033151)](https://hpo.jax.org/app/browse/term/HP:0033151)
- pmid: PMID:19001541
- def: Increase in the anterior posterior dimension of the prevertebral space, which is located in the neck region between the anterior part of the cervical spine and the deep layer of the deep cervical fascia running between the transverse processes of the spine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9019
- synonyms: Increased pre-vertebral space(NONE)
